src/main/java/jalau/at18/katas/bankocr/rodrigov/ReadFile.java

```diff
@@ -0,0 +1,41 @@
+package jalau.at18.katas.bankocr.rodrigov;
+import java.io.File;  // Import the File class
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+public class ReadFile {
+    private int cont = 0;
+    private final String path = "D:/AutomationTesting/Prog101/gitRemote/at18/src/main/java/jalau/at18/katas/bankocr/rodrigov/account.txt";
```

Comment:
This might not work for project, try using relative path, for example:
`src/main/java/jalau/at18/katas/bankocr/rodrigov/account.txt`

Reply:
No such file error fixed